### DISCUSSION TITLE
no aspect in my chart

### DISCUSSION THREAD

**Discussion by kiloppertry** [AstroDraw/AstroChart#67] [General]
![QQ截图20231129123711](https://github.com/AstroDraw/AstroChart/assets/39750125/4463abd8-1fe4-4ac3-92d1-6ed94a957703)
![QQ截图20231129123427](https://github.com/AstroDraw/AstroChart/assets/39750125/1823ee7f-7920-4125-ba16-703ce5936504)

i use react.js 

    const chart = new Chart("paper", 800, 800);
    chart.radix(data);

          <Box sx={{ position: "fixed", left: "5%", top: "5%" }}>
            <div id="paper"></div>
          </Box>

**Comment by kiloppertry** [comment]
and the sirius is a red point not a symbol

**Comment by afucher** [comment]
Hey @kiloppertry ,

To show the aspects you need to call `radix.aspects`, [here is the doc](https://astrodraw.github.io/api/radix#aspects)
Abou the red circle, is because the library still does not have the symbol that you want. So in this case it will show the red circle to you.
If you want, you can pass [a function to `CUSTOM_SYMBOL_FN` in the settings](https://github.com/AstroDraw/AstroChart/blob/a16fb94f20b7400301ab44a1a7e5ad68b2de3d11/project/src/settings.ts#L99) that will allow you to customize the symbol that you want. Also, feel free to contribute and add new more symbols.
